File: 3_single_inheritance.py
class Animal:
    def __init__(self, name):
        self.name = name 



class animalCharacteristics(Animal):
    def __init__(self, name, height, weight, color):
        self.height = height 
        self.weight = weight 
        self.color = color 
        super().__init__(name)


    def display(self):
        print(self.name, self.height, self.weight, self.color)





a1 = animalCharacteristics('Tiger', '2f', '40kg', 'stripeless snow white ')

a1.display()

File: 3_single_inheritance.cpp
#include<bits/stdc++.h>
using namespace std;
 
class Animal {
public:
    string name;
 
    
};

class AnimalCharacteristics : public Animal {
public:
    string height;
    string weight;
    string color;
 
    AnimalCharacteristics(string name, string height, string weight, string color)
    {   
        this->name=name;
        this->height=height;
        this->weight=weight;
        this->color=color;
    }
         

    void display() {
        cout<<name<<" "<<height<<" "<<weight<<" "<<color<<endl;
    }
};

int main() {
    AnimalCharacteristics a1("Tiger", "2f", "40kg", "stripeless snow white");
    a1.display();
    return 0;
}